### DISCUSSION TITLE
make sketches / Drawing

### DISCUSSION THREAD

**Discussion by yourmother9** [gsantner/markor#2240] [Ideas] [answered]
is it possible to make sketches in markor? 
if not, is this a idea that people like?

**Comment by harshad1** [comment]
Markor is first and foremost a plain text editor. All other features are addons and _must_ work on top of the plain text format.

We support taking images and editing images. That could let you do something like this with the right external apps

**Comment by gsantner** [comment, marked as answer]
Maybe this comes handy to you. https://github.com/FossifyOrg/Draw
